kubernetes/kserve/kserve_wrapper/__main__.py

```diff
@@ -100,5 +100,5 @@ def parse_config():
     ModelServer(
         registered_models=registeredModels,
         http_port=8080,
-        grpc_port=7070,
+        grpc_port=8081,
```

Comment:
8081 is rest management port,  not grpc port.

Reply:
KServe uses 8081 as default gRPC port. Since Torchserve has support to handle both inference and management API on same port, I am using 8081 for gRPC.